pcsx2/gui/Panels/SpeedhacksPanel.cpp

```diff
@@ -66,31 +66,30 @@ const wxChar* Panels::SpeedHacksPanel::GetEEcycleSliderMsg( int val )
 	return L"Unreachable Warning Suppressor!!";
 }
 
-const wxChar* Panels::SpeedHacksPanel::GetVUcycleSliderMsg( int val )
+const wxChar* Panels::SpeedHacksPanel::GetEECycleSkipSliderMsg( int val )
 {
 	switch( val )
 	{
 		case 0:
 		{
 			const wxColour DarkSeaGreen = wxColour(14, 158, 19);
-			m_msg_vustealer->SetForegroundColour(DarkSeaGreen);
-			return pxEt(L"0 - Disables VU Cycle Stealing.  Most compatible setting!");
+			m_msg_eeSkip->SetForegroundColour(DarkSeaGreen);
+			return pxEt(L"Disables EE Cycle Skipping. Most compatible setting.");
 		}
 		case 1:
 		{
-			m_msg_vustealer->SetForegroundColour(wxColour(L"Red"));
-			return pxEt(L"1 - Mild VU Cycle Stealing.  Lower compatibility, but some speedup for most games.");
+			m_msg_eeSkip->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"Mild EE Cycle Skipping. Lower compatibility, may cause variable framerate games to reduce internal framerate. Helps some games with VU starvation problems run at full speed. E.g. Shadow of the Colossus.");
 		}
 		case 2:
 		{
-			m_msg_vustealer->SetForegroundColour(wxColour(L"Red"));
-			return pxEt(L"2 - Moderate VU Cycle Stealing.  Even lower compatibility, but significant speedups in some games.");
+			m_msg_eeSkip->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"Moderate EE Cycle Skipping. Even lower compatibility, variable FPS games will reduce internal FPS. Games with VU starvation problems will gain emulation speed, others will likely start to slow down.");
 		}
 		case 3:
 		{
-			// TODO: Mention specific games that benefit from this setting here.
-			m_msg_vustealer->SetForegroundColour(wxColour(L"Red"));
-			return pxEt(L"3 - Maximum VU Cycle Stealing.  Usefulness is limited, as this will cause flickering visuals or slowdown in most games.");
+			m_msg_eeSkip->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"Maximum EE Cycle Skipping. Mostly harmful, variable FPS games will significantly reduce internal FPS. Games with VU starvation problems will gain emulation speed. Most games may slow to a crawl.");
```

Comment:
Ditto.

Also, I still don't really like the "slow down to a crawl" expression because I think it might be harder to understand for non-native English speakers (maybe it isn't an issue, but my first language is English so it's hard for me to judge).